core/tests/import_export_file_tests.rs

```diff
@@ -0,0 +1,109 @@
+mod integration_test;
+
+#[cfg(test)]
+mod import_export_file_tests {
+    use lockbook_core::model::state::temp_config;
+    use lockbook_core::service::import_export_service::ImportExportFileInfo;
+    use lockbook_core::service::test_utils::generate_account;
+    use lockbook_core::{
+        create_account, create_file, create_file_at_path, export_file, get_file_by_path, get_root,
+        import_file, write_document,
+    };
+    use lockbook_models::file_metadata::FileType;
+    use rand::Rng;
+    use uuid::Uuid;
+
+    #[test]
+    fn import_file_successfully() {
+        // new account
+        let config = &temp_config();
+        let generated_account = generate_account();
+        create_account(
+            config,
+            &generated_account.username,
+            &generated_account.api_url,
+        )
+        .unwrap();
+        let root = get_root(config).unwrap();
+
+        let tmp_dir = tempfile::tempdir().unwrap().path().to_path_buf();
+
+        let config_copy = config.clone();
+        let f = move |info: ImportExportFileInfo| {
+            get_file_by_path(&config_copy, info.lockbook_path.as_str()).unwrap();
+            assert!(info.disk_path.exists());
+        };
```

Comment:
I think it's clearer if variables are declared as close as possible to where they are first used (`root` and `f` aren't used until `import_file`.